kqueen/auth/ldap.py

```diff
@@ -17,6 +19,11 @@ def __init__(self, *args, **kwargs):
         if not hasattr(self, 'uri'):
             raise Exception('Parameter uri is required')
 
+        try:
+            ldap.initialize(self.uri)
+        except Exception:
+            logger.exception('Failed to initialize LDAP connection')
```

Comment:
you catch and log an exception, but the program flow go further, I suggest to reraise exception or not to catch it HERE at all